crates/extensions/c8y_mapper_ext/src/tests.rs

```diff
@@ -1947,6 +1948,57 @@ EOF
     assert_command_exec_log_content(ttd, expected_content);
 }
 
+#[ignore] // HELP!
```

Comment:
;-) A concrete question might be helpful to get some help.

Reply:
More seriously: https://github.com/thin-edge/thin-edge.io/pull/3144#discussion_r1786102656

Reply:
fixed in e38f515cac74323ade9e64c7af1099db6cc5642e